overlord/restart/restart_test.go

```diff
@@ -294,6 +294,83 @@ func (s *restartSuite) TestFinishTaskWithRestart(c *C) {
 	}
 }
 
+func (s *restartParametersSuite) TestProcessRestartForChangeClassic(c *C) {
```

Comment:
shouldn't these then be on the restartSuite?

Reply:
yea thank you for catching that, it seemed like I had forgotten to move several unit tests back to the restartSuite.